wiki/People/Mapping_Assistance_Team/en.md

```diff
@@ -6,16 +6,27 @@ tags:
+Shortly afterwards, open applications were held, allowing all members of the general mapping and modding community to apply for a position in the MAT. On the 26th of February, 2010, the first new batch of MAT members were added from these applicants,[^2] based on a collective evaluation from members of the BAT and existing MAT. Further application cycles were held regularly to scout for new members, as well as provide feedback to unsuccessful applicants.
 
-On the 4th of October, 2010, members of the MAT gained the ability to place [full bubble](/wiki/Modding/Bubble) and [bubble pop](/wiki/Modding/Bubble#bubble-pop) icons on beatmaps, bringing their roles and responsibilities more into line with their BAT colleagues. However, they still did not possess forum or in-game chat moderation tools, and focused solely on evaluating and promoting beatmaps.
+On the 13th of April, 2010, the proto-bubble was officially discontinued for use by the MAT.[^3] After this change, the MAT role would not have any direct crontol over beatmaps, and would serve more as a title given to exceptional modders.[^4]
 
-Following a gradual convergence of experience and responsibility between the two teams, on the 23rd of April, 2013, the MAT was dissolved, with all current members being promoted to the BAT and receiving the associated moderation tools and responsibilities.
+On the 4th of October, 2010, members of the MAT gained the ability to place [full bubble](/wiki/Modding/Bubble) and [bubble pop](/wiki/Modding/Bubble#bubble-pop) icons on beatmaps,[^5] bringing their roles and responsibilities more into line with their BAT colleagues. However, they still did not possess forum or in-game chat moderation tools, and focused solely on evaluating and promoting beatmaps.
+
+Following a gradual convergence of experience and responsibility between the two teams, on the 23rd of April, 2013, the MAT was dissolved,[^4] with all current members being promoted to the BAT and receiving the associated moderation tools and responsibilities.
+
+## References
+
+[^1]:  [Forum post announcing the creation of the MAT.](https://osu.ppy.sh/community/forums/topics/21856?n=1)
+[^2]:  [Forum post announcing the addition of the new members.](https://osu.ppy.sh/community/forums/topics/25314?n=1)
+[^4]:  [Forum post announcing the end of the MAT.](https://osu.ppy.sh/community/forums/topics/129165?n=1)
+[^3]:  [(Restricted) Forum post announcing the discontinuation of proto-bubbles for MATs.](https://osu.ppy.sh/community/forums/topics/28143?n=57)
+[^4]:  [(Restricted) Forum post discussing the state of the MAT.](https://osu.ppy.sh/community/forums/topics/36277?n=1)
+[^5]:  [Forum post announcing that MATs are able to bubble maps.](https://osu.ppy.sh/community/forums/topics/38405?n=1)
```

Comment:
a few things here:
- as CI says already, you have some duplicate definition IDs here, looks like a mistake
- it would probably be better to give these definitions names instead of just numbering them
- when linking to a specific post, use the link of form `https://osu.ppy.sh/community/forums/posts/{post}`
- no need to say "(Restricted)" (also one of them is public anyway)

Reply:
I fixed the duplicate IDs and removed "(Restricted)", but trying to change the format of the link broke them and they sent me to completely unrelated posts. Also, I think the definitions being numbered are fine, they're just there to cite the source, nothing fancy is need.